### PR TITLE
Ka 2018 03 add image fields to idea models

### PR DESCRIPTION
**Pull request by fuzzylogic2000**

### CODE REVIEW

File: meinberlin/apps/ideas/forms.py [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
 
 from . import models
 
 
 class IdeaForm(CategorizableFieldMixin, forms.ModelForm):
+    right_of_use = forms.BooleanField(required=False)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if self.instance.name != '' and self.instance.image:
```
Comment (MagdaN) [inline meinberlin/apps/ideas/forms.py]: I think you can check for `self.instance.pk` here, as the instance won't have a pk when it has not been saved

File: meinberlin/apps/mapideas/forms.py [hunk 2]
```diff
@@ -18,10 +17,26 @@ def __init__(self, *args, **kwargs):
             polygon=self.settings.polygon)
         self.fields['point'].error_messages['required'] = _(
             'Please locate your proposal on the map.')
+        if self.instance.name != '' and self.instance.image:
```
Comment (MagdaN) [inline meinberlin/apps/mapideas/forms.py]: the same here

File: meinberlin/apps/maptopicprio/forms.py [hunk 3]
```diff
@@ -20,10 +21,23 @@ def __init__(self, *args, **kwargs):
             polygon=self.settings.polygon)
         self.fields['point'].error_messages['required'] = _(
             'Please locate your proposal on the map.')
+        if self.instance.name != '' and self.instance.image:
```
Comment (MagdaN) [inline meinberlin/apps/maptopicprio/forms.py]: and here

File: meinberlin/apps/topicprio/forms.py [hunk 4]
```diff
@@ -10,7 +11,23 @@ class TopicForm(CategorizableFieldMixin, forms.ModelForm):
 
     description = fields.RichTextUploadingFormField(
         config_name='image-editor', required=True)
+    right_of_use = forms.BooleanField(required=False)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if self.instance.name != '' and self.instance.image:
```
Comment (MagdaN) [inline meinberlin/apps/topicprio/forms.py]: and here

File: meinberlin/apps/ideas/forms.py (hunk 1 above)
Comment (fuzzylogic2000) [reply meinberlin/apps/ideas/forms.py]: Yes, thanks! But even just checking for the image should be enough!

**Pull request merged**